redash/templates/invite.html

```diff
@@ -1,36 +1,39 @@
 {% extends "layouts/signed_out.html" %}
 
+{% block title %}Welcome to Redash!{% endblock %}
+
 {% block content %}
+<div class="fixed-width-page">
+  <div class="bg-white tiled">
+    <div class="m-b-default">
+      {% if google_auth_url %}
```

Comment:
Need to support SAML and the other providers here too (although SAML is the important one). How about, have two variations of the text:

Has SSO provider (Google, SAML, LDAP, etc) available: "To create your account, please choose a password or login with your SSO provider."

No SSO provider: "To create your account, please choose a password."

Then show the different login buttons as we do on the login page.